arborist/server_test.go

```diff
@@ -2643,6 +2649,15 @@ func TestServer(t *testing.T) {
 			})
 		})
 
+		t.Run("RevokePolicyThroughUser", func(t *testing.T) {
+			w := httptest.NewRecorder()
+			url := fmt.Sprintf("/user/%s/policy/%s", testGroupUser1, policyName)
+			req := newRequest("DELETE", url, nil)
+			req.Header.Add("X-AuthZ-Provider", "xxx")
+			handler.ServeHTTP(w, req)
+			assert.Equal(t, w.Code, http.StatusBadRequest, " Group policy must not be revoked directly through user")
```

Comment:
could we check that the policy is still in the user's policies, like the RevokePolicy test does?